process/datadog_checks/process/process.py

```diff
@@ -87,6 +135,13 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         # Process cache, indexed by instance
         self.process_cache = defaultdict(dict)
 
+        ProcessCheck.proc_list_cache_duration = int(init_config.get('proc_list_cache_duration',
```

Comment:
This needs to be documented in the configuration example? Maybe we could name it a bit better too? I don't have a great suggestion though.